src/pages/Options/index.css

```diff
@@ -28,3 +28,7 @@ header h2 {
   justify-content: center;
   align-items: center;
 }
+
+#app {
+  min-height: 600px;
```

Comment:
@teebszet 

This is the fix for the onboarding page. With min-height, there is the possibility of it extending past 600 if the contents are bigger, instead of changing over to scrollbar. I couldn't find a screen that's affected by it though.

An alternative would be to add a context provider in the layouts which sets the context to popup or full screen in the provider, and we can add a consumer on the screenContainer component which either sets the height to 600px or 100%, depending on the context?

Reply:
I think the simpler solution is better here. thanks for fixing!